### DISCUSSION TITLE
Sending sticker updates via webhooks

### DISCUSSION THREAD

**Discussion by mbrummerstedt** [python-telegram-bot/python-telegram-bot#4166] [Q&A]
I am trying to get webhook events sent to me using the bot.set_webhook() function, but I am not able to get it to send events when a sticker is added to a message. 

Is there a way to do it?

**Comment by Bibo-Joshi** [comment]
Hi. Please make sure that you have

* read [this wiki page](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Webhooks)
* pass `Update.ALL_TYPES` to [`allowed_updates`](https://docs.python-telegram-bot.org/en/stable/telegram.bot.html#telegram.Bot.set_webhook.params.allowed_updates) (or the approriate list of updates that you want to listen to)

Except for the `allowed_updates` there should be no reason for your bot not to get updates about a user sending a sticker-message
